daemon/seccomp_linux.go

```diff
@@ -26,7 +26,7 @@ func WithSeccomp(daemon *Daemon, c *container.Container) coci.SpecOpts {
 		if c.HostConfig.Privileged {
 			return nil
 		}
-		if !daemon.seccompEnabled {
+		if !daemon.sysInfo.Seccomp {
```

Comment:
note to self; this doesn't work; need to change this to use `RawSysInfo()` to fix one of the tests